src/main/java/org/folio/rest/impl/EholdingsAssignedUsersImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.rest.impl;
+
+import static io.vertx.core.Future.succeededFuture;
+
+import java.util.Map;
+import java.util.function.Function;
+
+import javax.ws.rs.core.Response;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.aspect.HandleValidationErrors;
+import org.folio.rest.jaxrs.resource.EholdingsKbCredentialsId;
+import org.folio.rest.util.ErrorHandler;
+import org.folio.service.assignedusers.AssignedUsersService;
+import org.folio.spring.SpringContextUtil;
+
+public class EholdingsAssignedUsersImpl implements EholdingsKbCredentialsId {
```

Comment:
`EholdingsKbCredentialsId` is not a good name. Any way to change it?

Reply:
Changed the name, thanks for review